en/docs/administer/admin-overview.md

```diff
@@ -0,0 +1,7 @@
+# WSO2 API Manager Administration Overview
+
+This section explains how to carry out the following administration activities for WSO2 API Manager.
+
+- [Applying WUM Updates]({{base_path}}/administer/admin-overview)
+- [Multitenancy]({{base_path}}/administer/multitenancy/introduction-to-multitenancy)
+- [Logging]({{base_path}}/administer/logging-and-monitoring/logging/setting-up-logging)
```

Comment:
There is a section on logging in the Observe section right now. Should all that content be here?

Reply:
Moved this section to Logging.